src/WebJobs.Script/Binding/NotificationHubScriptBindingProvider.cs

```diff
@@ -51,12 +51,8 @@ public override bool TryResolveAssembly(string assemblyName, out Assembly assemb
         {
             assembly = null;
 
-            if (string.Compare(assemblyName, "Microsoft.Azure.NotificationHubs", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
Okay, now I see it -- https://github.com/Azure/azure-webjobs-sdk-script/blob/ca785e62d5ad97bc810af33fed93ec16b660777c/src/WebJobs.Script/Description/DotNet/ExtensionSharedAssemblyProvider.cs#L36.